### ISSUE TITLE
[Process] Aligning with Relationship between groups document

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-charter-drafts#162]
There are overlaps between [the process document](https://github.com/w3c/wot-usecases/blob/main/Process.md) here and [the relationship document](https://github.com/w3c/wot-charter-drafts/blob/main/relationship.md) in the charter drafts repository.

We should make sure that they are aligned. The following points are already different:

1. In the relationship document, a use case can reside in any repository, like a CG, but they have to comply to the template set by the Use Cases TF. Use Cases TF is tasked with extraction of requirements.
2. Testing: Process document here states that each spec TF (collaboratively with the whole WoT-WG/IG) is in charge of the process. The relationship document states that any group (such as CGs) can be involved where they own the test cases that comply to the rules set by the TF.

**Comment by mmccool** [comment]
Someone needs to suggest a PR to fix this if we have inconsistent process documentation.
However, the "relationship" document is a draft and has now diverged from our actual process, so I think the process document HERE should take precedence and the "relationship" document should be updated.  Do you agree, @egekorkan 

At any rate, that makes this a charter-drafts issue so we should move it to that repo.

**Comment by egekorkan** [comment]
I would agree that the process document here should be followed. charter draft would be just the input for that. Given that the charter is expiring anyways, I am closing this.
